docker-compose-host.yml

```diff
@@ -44,6 +44,7 @@ services:
   lms-worker:
     volumes:
       - ${DEVSTACK_WORKSPACE}/edx-platform:/edx/app/edxapp/edx-platform
+      - ${DEVSTACK_WORKSPACE}/src:/edx/src
```

Comment:
Looks good, but do you also want to add it for cms-worker?

Reply:
Done.